core/geometry/src/test/clipping/ClipUtilities.test.ts

```diff
@@ -366,6 +367,157 @@ describe("ClipUtilities", () => {
       GeometryCoreTestIO.saveGeometry(allGeometry, "ClipUtilities", "ClipLoop2");
       expect(ck.getNumErrors()).equals(0);
     });
```

Comment:
This test is moved from "Polyface.test.ts" to "ClipUtilities.test.ts" with no change because here is a better place for it.